packages/build-info/src/build-systems/nrwl.ts

```diff
@@ -45,29 +85,32 @@ export class Nx extends BaseBuildTool {
           const { workspaceLayout } = await fs.readJSON<any>(fs.join(this.project.jsWorkspaceRoot, 'nx.json'))
           // if an apps dir is specified get it.
           if (workspaceLayout?.appsDir?.length) {
-            const identifyPkg: identifyPackageFn = async ({ entry, directory }) => {
-              if (entry === 'project.json') {
+            const identifyPkg: identifyPackageFn = async ({ entry, directory, packagePath }) => {
+              // ignore e2e test applications as there is no need to deploy them
+              if (entry === 'project.json' && !packagePath.endsWith('-e2e')) {
```

Comment:
Is this `-e2e` suffix an official convention? If so, can we maybe link to it in a comment? And if not, what's the impact of false positives?